### DISCUSSION TITLE
I always get rejected and i dont know why

### DISCUSSION THREAD

**Discussion by tm-frs** [community/community#59904] [GitHub Education]
### Hi everyone,

Hey guys, i have yesterday tried to apply, but I have a problem that I dont even know what it means. My school doesn't have email adresses for students, so i just chose my personal Gmail adress and I uploaded a photo of my student id. It shows the location of my school, its name, my name, my date of birth, a photo of me and the date of expiration (which is 09/2024, so that can'be the problem). When I clicked process, I immediatly got the following response:

> Unfortunately, we weren't able to approve your educational discount request for @timofreese.
>
> What happened?
> The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
> * If your school does not provide academic email then you must enable use of your device camera to capture your image.

I would be happy if I at least knew what the problem is. It says I should capture my image, but I uploaded my student id and selected that it is a student id. I think I'm too stupid, I hope someone can help me

Any idea on what I could do would be welcome

**Comment by samus-aran** [comment]
Hi @timofreese 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
